src/config/2020.json

```
@@ -212,6 +212,17 @@
       "website": "https://aagar.github.io/",
       "github": "AAgar",
       "twitter": "AAgar"
+    },
+     "adityapandey1998": {
+      "name": "Aditya Pandey",
+      "teams": [
+       "analysts",
```

Nit missed a spacing:

```suggestion
        "analysts",
```